ur_robot_driver/include/ur_robot_driver/hardware_interface.hpp

```diff
@@ -126,7 +121,6 @@ class URPositionHardwareInterface : public hardware_interface::SystemInterface
   void updateNonDoubleValues();
 
   HardwareInfo info_;
```

Comment:
Not needed anymore, [see](https://github.com/ros-controls/ros2_control/blob/04fa19b50e9e67cf7e6d7cc5c7ba000e435440f9/hardware_interface/include/hardware_interface/system_interface.hpp#L198)

```suggestion
```

Reply:
d4a1834